rfc/produce-message-size.md

```diff
@@ -0,0 +1,99 @@
+# Enhancing Fluvio to Message Sizes
+
+This RFC proposes modifications to Fluvio's handling of message sizes. 
+
+## Proposed Enhancements
+
+1. Handling Larger Messages than Batch Size
+
+If a single record exceeds the defined `batch_size`, Fluvio will process the record as a standalone request, ensuring that larger messages are not discarded or delayed. If the record does not exceed the `batch_size`, Fluvio will process the record as part of an already existing batch or create a new one if the batch is full.
```

Comment:
Hmmm, is this true? Customers are complaining of packets being dropped if they are larger than the batch size.

Reply:
I wrote the expectation, I'll rewrite it with what is doing now and what should be changed.

Reply:
not sure why this behavior is needed.  should move to enhancement if this doesn't applies to original problem

Reply:
this can be subsumed into exiting batching behavior.  if `batch_size` is exceed, then batch is flushed